server_tests/test/clientTests.js

```diff
@@ -830,11 +1030,16 @@ describe('RUST API TEST SUITE', function () {
 
   // GET OPTIONS
   describe('Testing /osc/commands/execute camera.getOptions endpoint', function () {
-    var sessionId;
+    if (isOSC1) {
+      var sessionId;
+    }
+
     var specifiedOptions = ['captureMode', 'exposureProgram', 'iso', 'shutterSpeed', 'aperture',
       'whiteBalance', 'exposureCompensation', 'fileFormat', 'exposureDelay',
       'sleepDelay', 'offDelay', 'hdr', 'exposureBracket', 'gyro', 'gps',
-      'imageStabilization', '_bublVideoFileFormat'];
+      'imageStabilization', '_bublVideoFileFormat'].concat(isOSC2 ? ['previewFormat',
+      'captureInterval', 'captureNumber', 'remainingVideoSeconds', 'pollingDelay', 'delayProcessing',
+      'clientVersion'] : []);
```

Comment:
Should we throw these options into some constants? Ideally they should probably live in the osc-client